source/framework/tools/src/TRestReflector.cxx

```diff
@@ -157,45 +157,39 @@ int TRestReflector::InitDictionary() {
         }
     }
 
-    if (type == "" || size == 0 || (cl == 0 && !is_data_type)) {
+    if (type == "" || size == 0 || cl == nullptr) {
         cout << "Error in CreateDictionary: object is zombie!" << endl;
         return -1;
     }
 
-    if (1) {
-        int pos = ((string)type).find("<");
+    int pos = ((string)type).find("<");
```

Comment:
Please, add protection in case `pos==std::string::npos`

Reply:
If not found, will `string.substr(0, std::string::npos)` cause error? I guess it just returns the original string.

Reply:
Good guess!

```
root [0] std::string str = "Hello";
root [1] str.substr(0, str.find("<"))
(std::basic_string) "Hello"
```